loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -323,6 +323,7 @@ module BABYLON.GLTF2 {
             }
 
             node.babylonMesh = new Mesh(node.name || "mesh" + node.index, this._babylonScene);
+            node.babylonMesh.hasVertexAlpha = true;

```

Comment:
always?

Reply:
Yes. If vertex alpha is not available, it will not use it. The name is a bit confusing.